includes/Stories_Renderer/Renderer.php

```diff
@@ -425,23 +436,37 @@ protected function get_single_story_classes() {
 		return apply_filters( 'web_stories_renderer_single_story_classes', $classes );
 	}
 
+	/**
+	 * Gets the single story container styles.
+	 *
+	 * @return string Style string.
+	 */
+	protected function get_container_styles() {
+		$story_styles  = $this->is_view_type( 'circles' ) ? sprintf( '--ws-circle-size:%1$dpx', $this->attributes['circle_size'] ) : '';
+		$story_styles .= $this->is_view_type( 'carousel' ) ? sprintf( '--ws-story-max-width:%1$dpx', $this->width ) : '';
+
+		/**
+		 * Filters the web stories renderer single story classes.
+		 *
+		 * @param string $class Single story classes.
+		 */
+		return apply_filters( 'web_stories_renderer_container_styles', $story_styles );
```

Comment:
Add a since to the phpdocs.

Reply:
Will add `@since 1.3.0` for other methods in the class as well which doesn't have `@since` added to their phpdoc.

Reply:
It is likely that it will land in 1.4.0 but we can change that later.